packages/manager/modules/metrics/src/details/translations/Messages_fi_FI.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
All `Messages_fi_FI.json` files has been removed.
See: https://github.com/ovh/manager/pull/4229

Regarding the recent merge from Cloud Repsac migration for both Metrics and CDA, it looks like we can also remove them. I will do that in a separate Pull Request.

Hope it will helps you to operate some rebase.

Reply:
See: #4280